local_sales/app/views/offers/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<p id="notice"><%= notice %></p>
```

Comment:
What do you think of moving this line to the [`application.html.erb`](https://github.com/BC-MAY-21-ROR/proyecto-proy-t02/blob/master/local_sales/app/views/layouts/application.html.erb) file to avoid duplicating it?